Icebnb/Icebnb/PRGRoundRangeSlider.swift

```diff
@@ -0,0 +1,273 @@
+//
+//  PRGRoundRangeSlider.swift
+//  PRGRoundSliderTest
+//
+//  Created by John Spiropoulos on 20/03/2017.
```

Comment:
I guess we're keeping those to give credit? not sure here

Reply:
Yes, it's for giving credit